app/controllers/moments_controller.rb

```diff
@@ -1,4 +1,5 @@
 # frozen_string_literal: true
+require 'groupdate'
```

Comment:
Sure this is necessary?

Reply:
will test it out, my groking of ruby importing is still pretty patchy atm